### DISCUSSION TITLE
Welcome to ham-radio-freedom Discussions!

### DISCUSSION THREAD

**Discussion by hcarter333** [hcarter333/ham-radio-freedom#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hcarter333** [comment]
Hey @arhonah 
can you go to the repository for the ham radio exams?

**Reply by arhonah** [reply]
hullo!

**Reply by hcarter333** [reply]
hey. this is a lot of work

**Reply by arhonah** [reply]
yes it is ( :

**Reply by hcarter333** [reply]
look at #14 please
